packages/react/src/components/InAppMessaging/hooks/useMessageProps/types.ts

```diff
@@ -0,0 +1,50 @@
+import { MessagePayloadStyle } from '@aws-amplify/ui-react-core';
+import React from 'react';
+
+// import { ImageDimensions } from '../useMessageImage';
+
+/**
+ * Resolved message types
+ */
+export interface MessageComponentStyles {
```

Comment:
These are the styles provided from pinpoint, right? Should we add a comment describing why these styles are the only ones that can be overwritten?

Reply:
These aren't all coming from Pinpoint (e.g. image and close button styles aren't coming from payload). I think they represent (almost?) everything that is displayed in a message component, however, so we are actually allowing a lot of overriding via styles. Maybe the comment can be shored up (admittedly, it was copied from the React Native version of this hook)